tools/paths-lookup.sql

```diff
@@ -7,6 +7,6 @@ from
   instance_questions iq
   join assessment_questions aq on (aq.id = iq.assessment_question_id)
   join questions q on (q.id = aq.question_id)
-  join pl_courses plc on (plc.id = q.course_id)
+  join pl_courses plc on (plc.id = q.course_id) -- leave it? I think this script is interested in the host course of the question
```

Comment:
This script was removed in https://github.com/PrairieLearn/PrairieLearn/pull/7521, it should go away once you get up to date with master.